README.md

```diff
@@ -95,11 +95,13 @@ changes to the rules will not take effect until the references are updated.
 
 Steps for updating the rules:
 
-1. Merge the necessary changes to this repository - alerts and tests.
+1. Merge the necessary changes to this repository - alerts, recording rules and tests.
 2. The
-[data plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/cicd/saas-rhtap-rules.yaml#L35)
-and the
+[data plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/cicd/saas-rhtap-rules.yaml#L35),
+the
 [control plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/cicd/saas-rhtap-rules.yaml#L49)
+and the
+[recording rules staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/cicd/saas-rhtap-rules.yaml#L63)
```

Comment:
Yes, I haven't added it since I removed from app-interface MR as well because  Yftach and Avi mentioned it to test on stage first and if everything works properly then add production deployment as well.

Reply:
Ok, LGTM then.